src/main/java/org/launchcode/water_garden_tour/models/dto/RegisterDTO.java

```diff
@@ -0,0 +1,68 @@
+package org.launchcode.water_garden_tour.models.dto;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class RegisterDTO {
+
+    @NotBlank
+    private String fname;
+
+    @NotBlank
+    private String lname;
+
+    @NotBlank
+    @Size(min=5, max=20, message="Invalid username")
+    private String username;
+
+    @NotBlank
+    @Size(min=5, max=20, message="Invalid password")
+    private String password;
+
+    @NotBlank
+    @Size(min=5, max=20, message="Invalid password")
+    private String verifyPassword;
+
+    public RegisterDTO() {}
+
+    public String getFname() {
+        return fname;
+    }
+
+    public void setFname(String fname) {
+        this.fname = fname;
+    }
```

Comment:
no need to do anything but just wanted to say remember we might not need everything to be public here. Sometimes we might not want getters and setters for everything just making you think. =)

Reply:
Thank you, good point.